chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/JTestDefinition.java

```diff
@@ -100,6 +104,17 @@ public Builder withQueryProvider(Iterable queryProvider) {
         }
 
         /**
+         * Optional: Sets load balancer (subtypes of {@link QueryPoolLoadBalancer}).
+         * Default is {@link RoundRobinLoadBalancer}
+         *
```

Comment:
is it ok if you set the default value in this class instead of in client classes after null check. for example,
`if (jTestDefinition.getLoadBalancer() == null) {`